enso/experiment/__init__.py

```diff
@@ -261,17 +300,19 @@ def _split_dataset(dataset, training_size):
         # Unfortunately it doesn't work as expected and locks test size to train size
         test_size = int(len(dataset) * TEST_SETUP["sampling_size"])
         if test_size + training_size > len(dataset):
-            raise ValueError(
+            print(test_size, training_size, len(dataset))
```

Comment:
You could roll this into the print statement below w/ format and keep it there tbh. Useful for debugging.